curator-client/src/main/java/org/apache/curator/utils/ZKPaths.java

```diff
@@ -154,6 +154,12 @@ public String getNode()
         {
             return node;
         }
+
+        @Override
+        public String toString()
+        {
+            return "PathAndNode [path=" + path + ", node=" + node + "]";
```

Comment:
Why not return `ZKPaths.makePath(path, node)`?

Reply:
Because I want to know what the node was and the path was. I already know what I passed in :)